liwords-ui/src/lobby/active_games.tsx

```diff
@@ -17,12 +24,44 @@ type Props = {
 };
 
 export const ActiveGames = (props: Props) => {
+  const { useState } = useMountedState();
+  const [sortByLexicon, setSortByLexicon] = useState(
+    localStorage.getItem('sortByLexicon')
```

Comment:
could we rename the item inside localStorage to `lobbyFilterByLexicon` just to give it a little more of a namespace?